src/wwwroot/services/settings.js

```diff
@@ -137,5 +138,18 @@
 
       return plansCache;
     }
+
+    function getCountries() {
```

Comment:
mmmm, no, I am thinking about something more like creating a new _resources service_ and encapsulate there all of our weird code related to query API and language issues. 

What do you think?